docs/user-guide.md

```diff
@@ -1,16 +1,18 @@
 # User guide
 
+- [FAQ](/docs/user-guide/faq.md) - frequently asked questions about using and configurating stylelint.
+- [Complementary Tools](/docs/user-guide/complementary-tools.md) - list of editor plugins, runners/loaders and formatters.
+
 ## Usage
 
 - [The CLI](/docs/user-guide/cli.md) - options, examples and exit codes for using the CLI.
 - [The Node API](/docs/user-guide/node-api.md) - options and examples for using the Node API.
-- [The PostCSS Plugin](/docs/user-guide/postcss-plugin.md) - how to use the PostCSS plugin, either via a runner like `gulp-postcss` or via the PostCSS Node API.
+- [The PostCSS Plugin](/docs/user-guide/postcss-plugin.md) - options and examples for using the PostCSS plugin.
 
 ## Configuration
 
 - [Configuration](/docs/user-guide/configuration.md) - how to configure stylelint.
-- [Rules](/docs/user-guide/rules.md) - a list of all the rules that are available to you, and an explanation of the naming convention.
-- [Example Config](/docs/user-guide/example-config.md) - an example config that you can use as a foundation for crafting your own config.
-- [Plugins](/docs/user-guide/plugins.md) - a brief introduction to plugins, followed by a list of what's available.
+- [Rules](/docs/user-guide/rules.md) - a list of all stylelint's rules and an explanation of the naming conventions and how rules work together.
```

Comment:
A little clearer I think like this:

`a list of all stylelint's rules, along with an explanation of the naming conventions and how rules work together.`